apps/system/index.html

```diff
@@ -1193,5 +905,22 @@ <h1 data-l10n-id="deviceMenu"></h1>
       <div id="bottom-panel" class="gesture-panel" data-z-index-level="gesture-panel"></div>
 
     </div> <!-- end of #screen -->
+
+    <script defer src="shared/js/settings_listener.js"></script>
```

Comment:
What's the reasoning for placing these scripts at the bottom? Since we have the ```defer``` attribute, I think it's cleaner for them to be defined in the ```<head>```. I think it's cleaner to read that way.